priv/repo/migrations/20190613223618_add_permissions_to_hubs.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Ret.Repo.Migrations.AddPermissionsToHubs do
+  use Ecto.Migration
+
+  def change do
+    alter table("hubs") do
+      add(:permissions, :integer)
```

Comment:
do we need to initialize these with the defaults?

Reply:
nice catch